mlc-llm/tutorial_extensions_to_more_model_variants.ipynb

```diff
@@ -1004,9 +1004,7 @@
       "cell_type": "code",
```

Comment:
<p>Thanks for modifying the other two notebooks as well! For the "extensions to more model" one, I think there might be more changes to make.</p><p> </p><p>Since we introduced <code>mlc_chat.ConvConfig</code> and <code>mlc_chat.ChatConfig</code>, I think we can    modify Step 2 accordingly. Currently, it is manually changing the mlc-chat-config.json, and now we can pass in a ChatConfig. Perhaps in this case:</p><pre class="ql-syntax" spellcheck="false">chat_config = mlc_chat.ChatConfig(conv_config=mlc_chat.ConvConfig(
  stop_str="\n\n", system=""))

chat_mod = mlc_chat.ChatModule(model=model_path,
lib_path=lib, chat_config=chat_config)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlc-ai/notebooks/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Feel free to merge this first, and I can iterate it again. By that time, we would also have the cuda prebuilts and links to the documentation (especially API references) that we can put in the notebook

Reply:
@CharlieFRuan do you want to just make the change in this PR? I added you as a collaborator,

Reply:
Oh right that works! I'll edit it tonight, thanks!